homeassistant/components/cast/media_player.py

```diff
@@ -483,6 +486,19 @@ async def async_play_media(self, media_type, media_id, **kwargs):
             media_type = sourced_media.mime_type
             media_id = sourced_media.url
 
+        # Handle plex
+        if media_id.startswith(PLEX_URI_SCHEME):
```

Comment:
this logic could live in `play_media` instead of `async_play_media` since it's synchronous.

Reply:
Good call, that was bothering me. 👍